modules/imgproc/src/hal_replacement.hpp

```diff
@@ -615,6 +615,21 @@ inline int hal_ni_integral(int depth, int sdepth, int sqdepth, const uchar * src
 #define cv_hal_integral hal_ni_integral
 //! @endcond
 
+/**
+   @brief Calculate medianBlur filter
+   @param depth Depths of source and destination image
+   @param src_data,src_step Source image
+   @param dst_data,dst_step Destination image
+   @param width,height Source image dimensions
+   @param cn Number of channels
+   @param ksize Size of kernel
+*/
+inline int hal_ni_medianBlur(int depth, const uchar* src_data, size_t src_step, uchar* dst_data, size_t dst_step, int width, int height, int cn, int ksize) { return CV_HAL_ERROR_NOT_IMPLEMENTED; }
```

Comment:
BTW, depth and cn:
- may be combined into "int type"
- or placed on closer parameters positions

(Anyway looks like this is grabbed from other functions design)